tests/test_mock_base_gcal.py

```diff
@@ -0,0 +1,92 @@
+"""
+This is a unit test to mock the base_gcal method for funtion to check holidays and weekend in qxf2_SCHEDULER
+"""
+import unittest, mock
+import os,sys
+sys.path.append(os.path.dirname(os.path.dirname(os.path.abspath(__file__))))
+from qxf2_scheduler.qxf2_scheduler import is_qxf2_holiday,is_weekend
+import datetime
+ 
+class Check_Is_qxf2_holiday(unittest.TestCase):
+    """
+    This class contains unit test for is_qxf2_holiday method in qxf2_scheduler module
+    """
+    #date that is a qxf2 holiday
+    date='03/04/2019'
+    
+    #Hardcoding the date in datetime format
+    time_mock=datetime.datetime.strptime(date,'%m/%d/%Y')
+    
+    #mocking gcal.process_date_string method
+    @mock.patch('qxf2_scheduler.qxf2_scheduler.gcal.process_date_string',return_value=time_mock)      
+
+    def test_is_qxf2_holiday(self, mockbase_gcal):
```

Comment:
Please select a better method name. There can be many tests for the is_holiday method. So naming something test_is_, holiday is not very descriptive.